src/renderer/components/commands-action-button.tsx

```diff
@@ -120,15 +121,27 @@ export const GistActionButton = observer(
       const octo = await getOctokit(appState);
       const { gitHubPublishAsPublic } = appState;
       const options = { includeDependencies: true, includeElectron: true };
-      const values = await window.ElectronFiddle.app.getEditorValues(options);
+      const defaultGistValues = await getTemplate(appState.version);
+      const currentEditorValues = await window.ElectronFiddle.app.getEditorValues(
+        options,
+      );
 
       try {
+        const gistFilesList = appState.isPublishingGistAsRevision
+          ? this.gistFilesList(defaultGistValues)
+          : this.gistFilesList(currentEditorValues);
+
```

Comment:
issue(blocking): When I was testing this PR out, the `package.json` being written to the gist to appear as a "revision", which seems wrong.

See https://gist.github.com/erickzhao/b4bba605a5a4af69fe57380c8470ed32/revisions